_posts/2019-03-26-k8s-network.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title:  "br1dging the Gap I: Working with tap devices in k8s"
+date:   2019-03-26
+tags: solo5 rumprun kubernetes cni oci
+author: Brandon Lum
+---
+
+
+This is the first of several blog posts in the "`br1`dging the Gap" series! The series of posts will talk about the various gaps between what containers expect (and how the orchestration system provides), and what nabla containers expect. This was a prominent theme in the writing of runnc, and we hope to be able to share a little about our design choices in runnc. 
```

Comment:
mention that this is the "semantic gap" (known to VM people).

Reply:
modified:
"
This is the first of several blog posts in the "`br1`dging the Gap" series! The series of posts will talk about the various gaps between what containers and their orchestration systems expect and what  **nabla containers expect (also known as the semantic gap).**
This was a prominent theme in the writing of runnc, and we hope to be able to share a little about   our design choices in runnc.
"

Reply:
i'm a little bit worried that some people might correlate introspection to the semantic gap in reference to containers. Do you think this is a cause for confusion?

Reply:
I removed it since it requires some context of your current paper. Perhaps let's defer this to the FS post where we can explain it better in depth!